config/routes.rb

```diff
@@ -56,4 +56,5 @@
 
   root 'home#index'
   resources :cards
+  post 'trainer/review', path: 'review'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.